framework/db/pgsql/QueryBuilder.php

```diff
@@ -17,6 +17,12 @@
  */
 class QueryBuilder extends \yii\db\QueryBuilder
 {
+    const INDEX_UNIQUE = 'unique';
+    const INDEX_B_TREE = 'btree';
+    const INDEX_R_TREE = 'rtree';
+    const INDEX_HASH = 'hash';
+    const INDEX_GIST = 'gist';
```

Comment:
these constants should have a phpdoc description. Not sure if they are needed at all, we could allso passing raw string to `createIndex()`.